client/src/components/common/LikeButton.tsx

```diff
@@ -2,47 +2,39 @@ import { Heart } from 'src/assets/icons';
 import COLORS from 'src/constants/colors';
 import styled, { css } from 'styled-components/macro';
 
-import api from 'src/api';
-import { useRecoilValue } from 'recoil';
-import { userState } from 'src/recoil/atoms/user';
-import { useNavigate } from 'react-router-dom';
+import { useState } from 'react';
+
 interface LikeButtonProps {
-  productId: number;
   isLike: boolean;
   onClick: () => void;
 }
 
-export default function LikeButton({
-  productId,
-  isLike,
-  onClick,
-}: LikeButtonProps) {
-  const user = useRecoilValue(userState);
-  const navigate = useNavigate();
-  const handleLikeButtonClick = (
-    e: React.MouseEvent<HTMLButtonElement, MouseEvent>
-  ) => {
-    e.preventDefault();
-    if (!user) {
-      return navigate('/login');
-    }
-    api.likeProduct(productId);
+export default function LikeButton({ isLike, onClick }: LikeButtonProps) {
```

Comment:
isLiked나 hasLiked 라는 네이밍도 괜찮을 것 같은데 어떤가요?
```suggestion
export default function LikeButton({ isLiked, onClick }: LikeButtonProps) {
```

Reply:
좋다고 생각해요 ㅎㅎ 다른 컴포넌트랑 동일하게 isLiked로 통일하겠습니다